%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% resume - one page, one column
% XeLaTeX Template
% Version 1.1 (09.05.2017)
%
% Author: knyte (https://github.com/knyte)
%
% Repository: https://github.com/knyte/resume
%
% This template is primarily designed for
% undergraduate EE/CE/CS students applying for
% industry roles in software engineering
% 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\documentclass[]{resume-knyte}
 
\begin{document}

%%%%%%%%%
% HEADER
%%%%%%%%%

% \makeheader{title/name}
\makeheader{Tony Kabilan Okeke}

%%%%%%%%%%%%
% SUBHEADER
%%%%%%%%%%%%

% argument to environment should match number of sites
\begin{subheader}{1}
    % leave third argument empty to specify no hyperlink
    \site{}{kabilan108.github.io}{https://kabilan108.github.io/}
    \site{}{GitHub/kabilan108}{https://kabilan108.github.io/}
    \site{}{Linkedin/kabilan108}{https://linkedin.com/in/kabilan108}
    \bfsite{}{tonykabilanokeke@gmail.com}{mailto:tonykabilanokeke@gmail.com}
    \site{}{Philadelphia, PA}{}\\
\end{subheader}

%%%%%%%%%%%%
% EDUCATION
%%%%%%%%%%%%
\begin{topic}{Education}
    % \entry{institution}{dates}{subtitle}{description}
    \edentry{Drexel University, Philadelphia PA}{Sep 2019 - Jun 2024}
    {B.S. \& M.S. in Biomedical Engineering \qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad\qquad~GPA: 4.00}
    {\vspace{-10pt}{Concentrations:} Bioinformatics, Neuroengineering, Bioimaging}
    %  Relevant Coursework: Bioinformatics, Applied Machine Learning, Deep Learning, Computational Neuroscience}
\end{topic}


%%%%%%%%%
% SKILLS
%%%%%%%%%
\begin{topic}{Skills}
    % \skillset{category}{members}
    \skillset{Programming}{
        Python, R, Shell, C++, MATLAB, SQL, AWK, PHP, JavaScript
    }
    \skillset{ML Frameworks}{
        TensorFlow, PyTorch, HF Transformers, Scikit-learn, LangChain, LlamaIndex
    }
    \skillset{Tools \& Packages}{
        Seurat, SquidPy, CellProfiler, FastAPI, Flask, React.js, Docker
    }
    \\
\end{topic}
\vspace{-5pt}

%%%%%%%%%%%%%
% EXPERIENCE
%%%%%%%%%%%%%
\begin{topic}{Experience}
    % \entry{role}{dates}{team/company}{description}
    \entry{Data Scientist (Bioinformatics)}{Apr 2022 -  Sep 2022,~~Apr 2023 - Sep 2023}
    {IVIVT Non-Clinical Safety, GSK - Collegeville PA}
    {
    \item Enhanced spatial transcriptomics (10X Visium) data analysis by introducing advanced software and machine learning models to improve quality control, visualizations, cell type annotations and accurate quantification of gene expression across different tissue types.
    \vspace{-5pt}

    \item Facilitated the integration of internally generated spatial transcriptomics (ST) data with publicly availbale single-cell data to enable comprehensive analysis and interpretation of ST data in conjunction with broader cellular information.
    \vspace{-5pt}

    \item Developed Python scripts to streamline the identification of off-target genes expressed in specific tissues by automating the import of tissue-specific gene expression data from public domains like NCBI and Bgee.
    \vspace{-5pt}

    \item Collaborated with the HESI eSTAR carcinogenicity working group to develop a neural network model that predicts molecular initiating events associated with liver carcinogenicity, leveraging rat transcriptomics studies.
    \vspace{-5pt}

    \item Utilized UMAP and t-SNE clustering methods on a comprehensive ChEMBL dataset to group HESI compounds by global embeddings, and developed machine learning models for biomarker identification within each cluster to assess carcinogenicity, enabling more targeted and insightful evaluations.
    \vspace{-5pt}

    \item Spearheaded the development of an interactive analytics and visualization platform for spatial transcriptomics data (GeoMx), supporting future acquisition plans and facilitating in-depth, interactive reporting.
    \vspace{-5pt}

    \item Developed an interactive web-app for performing statistical analyses and biomarker discovery on high-content imaging datasets using \textit{Dash} and \textit{Flask}; scaled computations using \textit{Dask} and deployed on \textit{RStudio Connect}.
    \vspace{-5pt}

    \item Implemented machine learning algorithms including Decision Trees, Random Forests, and SVM for biomarker discovery in high-content imaging datasets.
    \vspace{-5pt}

    \item Implemented various deep learning models (Noise2Void, Cellpose) to improve the performance of Cell Profiler image segmentation and feature extraction pipelines.
    \vspace{-5pt}
    }\\

    \entry{Computational Research Assistant}{Mar 2021 - Aug 2022}
    {Invenio Lab, Penn Medicine - Philadelphia PA}
    {
        \item Led rigorous analytical efforts resulting in four peer-reviewed publications, focusing on the immune response in COVID-19 patients, including comprehensive studies on serum markers, urinary biomarkers, and neurological impacts.
        \vspace{-5pt}

        \item Conducted extensive data analysis to unveil temporal changes in serum markers related to organ failure and anti-viral treatments, contributing to a deeper understanding of COVID-19 progression and patient management strategies.
        \vspace{-5pt}
    
        \item Developed Python and R scripts to automate ingestion, quality control, and analysis of data generated from ELISA and Next Generation Sequencing assays; performed differential methylation and pathway enrichment analyses on NGS data sets.
        \vspace{-5pt}
    }\\

    \entry{Research Assistant}{May 2020 - Jun 2021}
    {Zhou Lab, Children's Hospital of Philadelphia - Philadelphia PA}
    {
        \item Contributed to the development of the `SeSAMe` package which provides utilities to support the analysis of Infinium DNA methylation data sets.
        \vspace{-5pt}

        \item Developed tests to validate package functionalites against publicly available datasets from the Gene Expression Omnibus (GEO).
        \vspace{-5pt}
    }\\
\end{topic}

%%%%%%%%%%%
% PROJECTS
%%%%%%%%%%%
\begin{topic}{Projects}
    % \entry{name}{dates}{tools used}{description}
    \entry{Meddibia}{\emph{Python, TensorFlow, Flask, Flutter}}
    {Mar 2023}
    {
        \item Pioneered a cross-functional team to win a \$5,000 prize at a hackathon, leveraging diverse skills to develop and deploy a medical diagnostic tool with a user-friendly mobile app interface.
        \vspace{-5pt}
    
        \item Engineered Python scripts for fine-tuning state-of-the-art neural networks like VGG16 and EfficientNet, achieving a 70\% classification accuracy for skin lesion analysis from Kaggle datasets using Google Colab for computation.
        \vspace{-5pt}
    
        \item Integrated GPT-3 for natural language processing to distill symptoms from patient descriptions, coupled with a Naive Bayes  classifier for disease prediction, and encapsulated the ML pipeline within a Flask API for real-time mobile app queries.
        \vspace{-5pt}
    }\\

    \entry{MLGO: Machine Learning for Predicting GO Enrichment}{\emph{Python, R, PyTorch, limma}}
    {Sep 2022}
    {
        \item Orchestrated an end-to-end R pipeline for preprocessing and differential expression analysis of over 11,000 RNA sequencing datasets from DEE2, leveraging Google Cloud VMs for high-compute capacity.
        \vspace{-5pt}

        \item Developed and tuned an autoencoder neural network to reconstruct log fold-change values from the datasets, explored latent space with PCA, UMAP, and t-SNE, and predicted Gene Ontology (GO) enrichment, utilizing tools such as TensorBoard and Keras Tuner for optimization.
        \vspace{-5pt}
    }\\

    \entry{Identification of Connectomic Biomarkers for Autism using Machine Learning}{\emph{Python, sklearn, bctpy}}
    {Aug 2022}
    {
        \item Developed Python scripts to compute a collection of graph theory measures on a dataset of structural connectomes from patients with Autism Spectrum Disorder (ASD); Employed machine learning models (LASSO regression, Support Vector Machine) for feature selection and diagnostic accuracy assessment.
        \vspace{-15pt}

        \item Executed a comprehensive study correlating connectomic biomarkers with clinical outcomes such as autism severity, social communication, and intelligence, utilizing data visualization tools to represent the intricate relationships within a multimodal neuroimaging dataset.
        \vspace{10pt}
    }
\end{topic}

%%%%%%%%%%%%%%%
% PUBLICATIONS
%%%%%%%%%%%%%%%
\begin{topic}{Publications}
    \edentry{}{}{}{
    \vspace{-35pt}
    Laudanski, K., Liu, D., \textbf{Okeke, T.}, Restrepo, M. and Szeto, W.Y., 2022. \textit{Persistent Depletion of Neuroprotective Factors Accompanies Neuroinflammatory, Neurodegenerative, and Vascular Remodeling Spectra in Serum Three Months after Non-Emergent Cardiac Surgery}. \href{https://doi.org/10.3390/biomedicines10102364}{Biomedicines 10.10 (2022): 2364}.
    }

    \edentry{}{}{}{
    \vspace{-30pt}
    Laudanski, K., \textbf{Okeke, T.}, Siddiq, K., Hajj, J., Restrepo, M., Gullipalli, D. and Song, W.C., 2022. \textit{A disturbed balance between blood complement protective factors (FH, ApoE) and common pathway effectors (C5a, TCC) in acute COVID-19 and during convalesce.} \href{https://doi.org/10.1038/s41598-022-17011-7}{Scientific Reports 12.1 (2022): 13658.}
    }

    \edentry{}{}{}{
    \vspace{-30pt}
    Laudanski, K., \textbf{Okeke, T.}, Hajj, J., Siddiq, K., Rader, D.J., Wu, J. and Susztak, K., 2021. \textit{Longitudinal urinary biomarkers of immunological activation in covid-19 patients without clinically apparent kidney disease versus acute and chronic failure.} \href{https://www.nature.com/articles/s41598-021-99102-5.pdf?proof=tr}{Scientific Reports, 11(1), p.19675.}
    }

    \edentry{}{}{}{
    \vspace{-30pt}
    Laudanski, K., Hajj, J., Restrepo, M., Siddiq, K., \textbf{Okeke, T.} and Rader, D.J., 2021. \textit{Dynamic changes in central and peripheral neuro-injury vs. Neuroprotective serum markers in COVID-19 are modulated by different types of anti-viral treatments but do not affect the incidence of late and early strokes.} \href{https://doi.org/10.3390/biomedicines9121791}{Biomedicines 9.12 (2021): 1791.}
    }

    \edentry{}{}{}{
    \vspace{-30pt}
    Laudanski, K., Jihane, H., Antalosky, B., Ghani, D., Phan, U., Hernandez, R., \textbf{Okeke, T.}, Wu, J., Rader, D. and Susztak, K., 2021. \textit{Unbiased analysis of temporal changes in immune serum markers in acute COVID-19 infection with emphasis on organ failure, anti-viral treatment, and demographic characteristics.} \href{https://www.frontiersin.org/articles/10.3389/fimmu.2021.650465/full}{Frontiers in Immunology, 12, p.650465.}
    }
    \\
\end{topic}
% \vspace{-10pt}

%%%%%%%%%%%%%%%
% CONFERENCE SUMBISSIONS
%%%%%%%%%%%%%%%
\begin{topic}{Abstracts}
    \edentry{}{}{}{
    \vspace{-40pt}
    \textbf{Okeke, T.}, Siddiq, K., Restrepo, M., Tadikonda, P. and Laudanski, K., 2022. 181: PATTERNS OF URINARY BIOMARKERS OF IMMUNOLOGIC ACTIVATION AND SEPSIS IN PATIENTS WITH COVID-19. \href{https://doi.org/10.1097/01.ccm.0000807048.74105.59}{Critical Care Medicine, 50(1), p.74.}
    }

    \edentry{}{}{}{
    \vspace{-60pt}
    Restrepo, M., \textbf{Okeke, T.}, Siddiq, K., Tadikonda, P. and Laudanski, K., 2022. 174: LONGITUDINAL CHANGES OF NEURO-SPECIFIC SERUM PROTEINS IN COVID-19 PATIENTS. \href{https://doi.org/10.1097/01.ccm.0000807020.73561.ed}{Critical Care Medicine, 50(1), p.71.}
    }
\end{topic}
\vspace{-20pt}

%%%%%%%%%
% HONORS
%%%%%%%%%
\begin{topic}{Honors}
    \honor{Drexel University Dean's List}{Fall 2019 - Present}
    \honor{HESI GTTC Professional Development Award, \$1000}{Spring 2023}
    \honor{Philly CodeFest Collaborative Team Award (Meddibia), \$5000}{Winter 2023}
\end{topic}
\vspace{5pt}

%%%%%%%%%
% SERVICE
%%%%%%%%%
\begin{topic}{Service \& Leadership}
    % \honor{award}{date/location}
    \honor{Drexel Computational Design, Vice President \& Cofounder}{Mar 2021 - Present}
    \honor{Drexel Society of AI, Member}{Sep 2022 - Present}
    \honor{Mid-Atlantic SOT, Member}{Oct 2023 - Present}
    \honor{Tau Beta Pi, Member}{Dec 2021 - Present}
\end{topic}

\end{document}
